### PR TITLE
initial commit for application page

### PR DESCRIPTION
**Pull request by torchiad**
Still using mocked data, types are in place though

### CODE REVIEW

File: src/components/Application/Application.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                                                as={RouterLink}
+                                                to={`/question/${question.id}`}
+                                            >
+                                                {question.heading}
+                                            </Link>
+                                        </Table.Cell>
+                                        <Table.Cell>
+                                            {question.complete
+                                                ? "Complete"
+                                                : "incomplete"}
```
Comment (torchiad) [inline src/components/Application/Application.tsx]: capitalise and add icon

File: src/components/Application/Application.test.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { Application } from "./index";
+
+describe("Application", () => {
+    it("renders correctly", () => {
+        const tree = renderer
+            .create(<Application application={() => ""} />)
```
Comment (ascorbic) [inline src/components/Application/Application.test.tsx:8]: ??

File: src/components/Application/Application.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+            <LoadingBox loading={!application}>
+                <Caption mb={1}>{application.opportunityName} </Caption>
+                <Title mb={5}>Application form</Title>
+
+                <ApplicationSummary key={application.id}>
+                    <GridRow>
+                        <GridCol setWidth="33%">
+                            <GridRow>
+                                <strong>Application number:</strong>
```
Comment (ascorbic) [inline src/components/Application/Application.tsx]: You know this is wrong! Use the govuk components.

File: src/components/ApplicationPage/ApplicationPage.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+import { getFundingApplication } from "../../graphql/queries";
+import { useQuery } from "react-apollo-hooks";
+import gql from "graphql-tag";
+
+interface Props extends RouteComponentProps {}
+
+const GET_APPLICATION = gql(getFundingApplication);
+
+const v: GetFundingApplicationQuery = {
```
Comment (ascorbic) [inline src/components/ApplicationPage/ApplicationPage.tsx]: There's no reason to use mock data in the live page. Mock it in the storybook to check designs, and use live data in the real site.

File: src/components/AuthWrapper/AuthWrapper.tsx [hunk 5]
```diff
@@ -39,6 +40,8 @@ export const AuthWrapper: FC<Props> = props => {
                     <ConfirmSignUp authProps={props} path="/confirm" />
                 )}
                 <ConfirmSignUp authProps={props} path="/confirm/:code" />
+
+                <ApplicationPage path="/application/:id" />
```
Comment (ascorbic) [inline src/components/AuthWrapper/AuthWrapper.tsx]: This doesn't belong here. This component is for wrapping all the authentication stuff.

**Review by ascorbic — CHANGES_REQUESTED** [review]

**Comment by torchiad** [comment]
@ascorbic the last commit fixes the issue I had with the AuthWrapper Component forcing the redirect. I believe these are all the points mentioned above covered.

**Review by ascorbic — APPROVED** [review]

**Pull request merged**